### DISCUSSION TITLE
Can we add tailwind class inside SVG file?

### DISCUSSION THREAD

**Discussion by riolly** [tailwindlabs/tailwindcss#10474] [Help]
I want to write the tailwind class in the SVG file  & import it rather than copy the inline style into my project file (Astro).

Here is my config file
```
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: ['./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue,svg}''],
  theme: {},
  plugins: [],
}
```
I add the SVG format to the content.

But somehow the class autocompletes don't show up & it doesn't take any effect if I add one.
[Here is the screenshot](https://www.linkpicture.com/q/Screenshot-from-2023-02-03-09-23-49.png)

**Comment by riolly** [comment]
For now, my current hack is to rename it to .astro
I think it's a pretty good hack.

**Comment by mrcaidev** [comment]
Your svg file is located in `/public`, but you only specified `/src` in your `content` config.

**Reply by mrcaidev** [reply]
Also, if you want to use it like `<img src={someSvg} alt="" />`, then Tailwind classes would not take effect (although they are still generated I guess), because the svg is not inlined in your HTML.

**Reply by riolly** [reply]
I put it on src/assets.
Still doesn't work.

The problem renames it with `.astro` is it's inlined with the HTML.
It's not a problem for now. 
Still, kinda think the case when the SVG is large that I want to bundle it separately,

**Reply by mrcaidev** [reply]
If you want to use Tailwind classes, it's inevitable to inline it in HTML. At least imho.
